src/ng-dashboard-aggregator/values.yaml

```diff
@@ -0,0 +1,151 @@
+# Default values for ng-dashboard-aggregator
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
+  loadbalancerURL: ""
+  database:
+    postgres:
+      ## - installed = true if installed within cluster
+      installed: true
+      ## - protocol to use for connection
+      protocol: postgres
+      ## - host array for external
+      hosts:
+        - postgres:5432
+      ## - secret name containing external values
+      secretName: ""
+      ## - key within secret containing username
+      userKey: ""
+      ## - key within secret containing password
+      passwordKey: ""
+      ## - extra arguments set to connection string
+      extraArgs: ""
+    mongo:
+      installed: true
+      protocol: mongodb
+      # --  provide default values if mongo.installed is set to false
+      hosts: []
+      secretName: ""
+      userKey: ""
+      passwordKey: ""
+      extraArgs: ""
+    timescaledb:
+      installed: true
+      protocol: "jdbc:postgresql"
+      # --  provide default values if mongo.installed is set to false
+      hosts:
+        - timescaledb-single-chart:5432
+      secretName: ""
+      userKey: ""
+      passwordKey: ""
+      extraArgs: ""
+  license:
+    cg: ''
+    ng: ''
+  ccm:
+    enabled: false
+  opa:
+    enabled: false
+  chaos:
+    enabled: false
+  stackDriverLoggingEnabled: false
+replicaCount: 1
+maxSurge: 1
+maxUnavailable: 0
+
+appLogLevel: "INFO"
+
+image:
+  registry: docker.io
+  repository: harness/ng-dashboard-aggregator-signed
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "700243-000"
+  digest: ""
+  imagePullSecrets: []
+
+ceGcpSetupConfigGcpProjectId: "placeHolderGcpProjectId"
+
+timescaleSecret:
+  password:
+    name: harness-secrets
+    key: timescaledbPostgresPassword
+
+java:
+  memory: "768m"
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "harness-default"
+
+podAnnotations: {}
+
+imagePullSecrets: {}
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+securityContext:
+  runAsUser: 65534
+  runAsNonRoot: true
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+probes:
+  readinessProbe:
+    httpGet:
+      path: /health
+      port: dashbrd-port
+    initialDelaySeconds: 200
```

Comment:
that is a very high initialDelaySeconds, do we require that much time?
we could use an initContainer(waiting for ng-manager and then set a very low initialDelaySeconds)

Reply:
Fixed